app/mailers/notify_mailer.rb

```diff
@@ -1,6 +1,6 @@
 class NotifyMailer < ApplicationMailer
   SUBJECTS = {
-    new_follower_email: "just followed you on dev.to".freeze
+    new_follower_email: "just followed you on #{ApplicationConfig['COMMUNITY_NAME']}".freeze
```

Comment:
I've changed dev.to to be referenced as DEV instead as this now becoming standardized across the site.